charts/internal/shoot-control-plane/templates/network-policies.yaml

```diff
@@ -44,6 +44,19 @@ spec:
 ---
 apiVersion: metal-stack.io/v1
 kind: ClusterwideNetworkPolicy
+metadata:
+  name: allow-only-apiserver-https
+  namespace: firewall
+spec:
+  egress:
+  - toFQDNs:
+    - cidr: kubernetes.default.svc
```

Comment:
We don't need this rule, and I'll explain why:
- CoreDNS runs on the worker nodes of a cluster and is not part of the control-plane
- internal DNS requests within the cluster (*default.svc) will be resolved by CoreDNS and other names will be looked up with an external resolver (CoreDNS asks the resolver of the operating system, systemd-resolved, which will on cache misses ask a public DNS server)
- only those requests that CoreDNS can't answer and systemd-resolved hasn't cached, will be sent over the firewall
- there is already CWNP allowing traffic exactly to the IP of the kube-apiserver in the control-plane belonging to this shoot cluster